src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -200,7 +200,8 @@ def assign_to_checkpoint(
         # Global renaming happens here
         new_path = new_path.replace("middle_block.0", "mid_block.resnets.0")
         new_path = new_path.replace("middle_block.1", "mid_block.attentions.0")
-        new_path = new_path.replace("middle_block.2", "mid_block.resnets.1")
+        new_path = new_path.replace("middle_block.2", "mid_block.temp_attentions.0")
+        new_path = new_path.replace("middle_block.3", "mid_block.resnets.1")
```

Comment:
I don't fully understand what this is used for. Is this a breaking change for previous Stable Diffusion converters?

Reply:
Ups thanks for spotting!